tf2_ros_py/tf2_ros/buffer_client.py

```diff
@@ -65,7 +66,7 @@ class BufferClient(tf2_ros.BufferInterface):
     def __init__(
         self,
         node: Node,
-        ns: str,
+        ns: str = "tf2_buffer_server",
```

Comment:
We can always create a `PixelBufferClient` which sets that attribute, in case this shouldn't be modified upstream.
As our python server currently handles the load just fine, I think a single buffer server in cpp should suffice for now

Reply:
> As our python server currently handles the load just fine

mmm but it handles pyro requests fine; service calls overload might be different? We'll see